src-java/base-topology/base-messaging/src/main/java/org/openkilda/messaging/info/switches/v2/LogicalPortsValidationEntryV2.java

```diff
@@ -59,6 +60,8 @@ static LogicalPortsValidationEntryV2 join(List<LogicalPortsValidationEntryV2> en
         builder.proper(joinLists(nonNullEntries.stream().map(LogicalPortsValidationEntryV2::getProper)));
         builder.missing(joinLists(nonNullEntries.stream().map(LogicalPortsValidationEntryV2::getMissing)));
         builder.misconfigured(joinLists(nonNullEntries.stream().map(LogicalPortsValidationEntryV2::getMisconfigured)));
+        nonNullEntries.stream().filter(e -> StringUtils.isNoneBlank(e.error)).findFirst()
```

Comment:
Why `findFirst`? What if there more than one element in the steam?

Reply:
Thanks, I thought about it, but actually if there will be multiple elements, then they have to contain the same error status(all the same, in case of null, all should contain null), due to the way how this collection filled initially. Otherwise it should be error probably, because if this field will be different among multiple components it means that some problems during this collection creation. 
We have to think about this field that spread across the whole collection as a single value. (whether this api design mistake or not, I am not sure.)
Could you please suggest the best way to deal with this kind of situation?
.. Or we could leave it as it is?

Reply:
Seems like there are a few assumptions (which might not be true) and that makes it more difficult to understand the code. The behavior doesn't have to be deterministic (execute the same steps twice, but receive different errors). So, I wouldn't leave it as it is. 
At least you can log the case: "Otherwise it should be error probably". Another option is to remove dupes from the stream of errors (for example, Stream::distinct) and then use Collectors::joining to keep all errors. If there is really only the duplicate errors, you get the same result as it is now, nothing to lose. But if, for whatever reason, it is not, you don't swallow errors here. To me that makes sense, I'd suggest this approach. The whole thing is a join method, so you join the errors as well. Yet another option is to change String to a collection of strings, but this is tricky because it might be a breaking change and probably is out of scope for this task.

Reply:
Thank you, fixed, please check.
@dmitrii-beliakov